web/src/beta/features/Editor/Tabs/Storytelling/StoryPageIndicator/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+import StoryPageIndicatorItem from "./StoryPageIndicatorItem";
+
+type Props = {
+  currentPage: number;
+  currentPageProgress: number;
+  maxPage: number;
+  onChangePage: (page: number) => void;
+};
+
+const StoryPageIndicator: FC<Props> = ({
+  currentPage,
+  currentPageProgress,
+  maxPage,
+  onChangePage,
```

Comment:
As stated in the [style guide](https://github.com/reearth/guides/blob/main/frontend/style.md) please make sure when possible to name prop functions `onNounVerb`, like onProjectCreate, onWorkspaceDelete, etc 👍🏻

Reply:
Thank you! Fixed in https://github.com/reearth/reearth/pull/543/commits/163e32207070d4421d00b2f2cf2db68571846952